specs.md

```diff
@@ -0,0 +1,138 @@
+Statements for imperative proofs (SOLAR-style)
+====================
```

Comment:
Formatting. Also, call this evm-imp-specs.md.

Reply:
The formatting should have the `===` extend the full length of the previous line.